arbotix_python/src/arbotix_python/publishers.py

```diff
@@ -81,7 +81,7 @@ def update(self, joints, controllers):
             msg.velocity = list()
             for joint in joints:
                 msg.name.append(joint.name)
-                msg.position.append(joint.position)
+                msg.position.append(joint.jst_position)
```

Comment:
I've tried your changes since I also have a phantom x pincher arm, but it seems like *jst_position* is not filled in adequately for each joint. For all arm joints the *jst_position* value was zero for me. With the small "hack":

    if joint.jst_position==0:
      joint.jst_position = joint.position

everything worked, but this actually is not a good solution ;-)